pkg/monitor/monitor.go

```diff
@@ -264,12 +264,12 @@ func (monitor *agentMonitor) getAgentCRD(crdName string) *v1beta1.AntreaAgentInf
 	return agentCRD
 }
 
-func (monitor *agentMonitor) createAgentCRD(crdName string) (*v1beta1.AntreaAgentInfo, error) {
+func (monitor *agentMonitor) GetAgentInfo(name string) *v1beta1.AntreaAgentInfo {
```

Comment:
I don't see the argument `name` is necessary, it's kept as `name` field of `agentMonitor`, why we are passing it and have to make clients fetch the name via another method first? So do `createAgentCRD`

Reply:
Updated.